backend/src/main/java/com/redhat/cloud/notifications/routers/internal/kessel/KesselAssetsMigrationService.java

```diff
@@ -115,7 +123,7 @@ public void onCompleted() {
 
     @Path("/kessel/migrate-assets/async")
     @POST
-    public void migrateAssetsAsync() {
+    public void migrateAssetsAsync(final Optional<String> orgId) {
```

Comment:
Any reason why `migrateAssets` and `migrateAssetsAsync` have a different kind of argument?